test/services/SaveMessageServiceSpec.scala

```diff
@@ -141,7 +141,7 @@ class SaveMessageServiceSpec extends SpecBase with BeforeAndAfterEach {
     }
 
     "return Failure when we cannot parse the message due malformed time" in {
-      when(mockXmlValidationService.validate(any(), any())).thenReturn(Success())
+      when(mockXmlValidationService.validate(any(), any())).thenReturn(Success(()))
```

Comment:
is there a reason for adding in the unit here? xmlValidationService wasn't touched

Reply:
To resolve compiler warnings

Reply:
To resolve compiler warnings